pkg/sql/logictest/testdata/logic_test/ddl_analysis

```diff
@@ -0,0 +1,23 @@
+# LogicTest: !3node-tenant
```

Comment:
Have you run this test under `stress` to ensure that the data you get is consistent?

Reply:
Ran it under stress for 10 mins, it is consistent.